src/Limit/CallbackLimit.php

```diff
@@ -11,11 +11,19 @@ public function __construct(callable $callback)
         $this->callback = $callback;
     }
 
-    public function __invoke() : bool
+    public function hasBeenReached() : bool
     {
         return call_user_func($this->callback);
     }
 
+    /**
+     * @deprecated  use hasBeenReached() instead
+     */
+    public function __invoke() : bool
+    {
+        return $this->hasBeenReached();
```

Comment:
Add in something like `@trigger_error('Using CallbackLimit as a callable is deprecated. Use CallbackLimit:: hasBeenReached() instead.', E_USER_DEPRECATED);`